### PR TITLE
Small minor performance improvements

### PR DESCRIPTION
**Pull request by TheRealHaui**
Small minor performance improvements.

### CODE REVIEW

**Comment by TheRealHaui** [comment]
I completely agree with you that measuring will be very complicated.
However: https://stackoverflow.com/questions/1508975/why-is-list-size0-slower-than-list-isempty-in-java

File: metrics/prometheus/src/main/java/io/helidon/metrics/prometheus/PrometheusSupport.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017, 2018 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
  *
```
Comment (barchetta) [inline metrics/prometheus/src/main/java/io/helidon/metrics/prometheus/PrometheusSupport.java]: Can you change this to `2017, 2019"? Thanks.

Comment (TheRealHaui) [reply metrics/prometheus/src/main/java/io/helidon/metrics/prometheus/PrometheusSupport.java]: Done.

**Comment by barchetta** [comment]
Thanks for contributing!

**Comment by TheRealHaui** [comment]
You are welcome!

**Pull request merged**